controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -74,6 +74,7 @@ func (obj *ocsNoobaaSystem) ensureCreated(r *StorageClusterReconciler, sc *ocsv1
 			Labels: nbv1.LabelsSpec{
 				"monitoring": getNooBaaMonitoringLabels(*sc),
 			},
+			DisableLoadBalancerService: sc.Spec.MultiCloudGateway.DisableLoadBalancerService,
```

Comment:
Because sc.Spec.MultiCloudGateway is a pointer, it can be null, causing tests to fail. This should be moved to the'setNooBaaDesiredState' function. Could you please relocate it?